### PR TITLE
CB-16438.  Configure and start SaaS related VM monitoring services by CB

### PR DESCRIPTION
**Pull request by oleewere**
See detailed description in the commit message.

### CODE REVIEW

File: common-model/src/main/java/com/sequenceiq/common/api/telemetry/model/Monitoring.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.sequenceiq.common.api.telemetry.model;
+
+import java.io.Serializable;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Monitoring implements Serializable {
```
Comment (horadla23) [inline common-model/src/main/java/com/sequenceiq/common/api/telemetry/model/Monitoring.java:8]: why do we have `Monitoring` and `MonitoringBase` with same field?

**Comment by horadla23** [comment]
Some initial questions:
- Is there a way to test this whole implementation somehow? With e2e probably? If there is a bug somewhere in this PR, it is hard to catch by looking at the code.
- Can you check why swagger check failed?

File: common-model/src/main/java/com/sequenceiq/common/api/telemetry/model/Monitoring.java (hunk 1 above)
Comment (oleewere) [reply common-model/src/main/java/com/sequenceiq/common/api/telemetry/model/Monitoring.java:8]: one of it is the json that is persisted in the DB, other one is for the API, these are not always strongly the same and can change/shift from each other, now at the moment the available configs are minimal, but will be extended later

**Comment by oleewere** [comment]
> Some initial questions:
> 
> * Is there a way to test this whole implementation somehow? With e2e probably? If there is a bug somewhere in this PR, it is hard to catch by looking at the code.
> * Can you check why swagger check failed?
- no prometheus endpoint yet for e2e tests, also some of the endpoints (mow-priv/dev etc.) can change at the moment, probably the global configs will be used for the testing, but make sense to clear out how the target environments will look like.
on the other hand, this feature should not be used yet (until it's not finished), so if cluster deployment still working fine as expected, that should be good enough, on next steps (by implementing CM based metrics exports etc.) the feature can be enabled which can include e2e test changes. so TLDR: e2e tests will be implemented later when these features will be a bit more ahead (until that the models/api will be available + will be only enabled with set some local configs)
- i think here a rebase could solve the swagger issue

**Review by horadla23 — APPROVED** [review]

**Pull request merged**